example_test.go

```diff
@@ -8,41 +8,53 @@ package fsnotify_test
 
 import (
 	"log"
+	"os"
+	"os/signal"
+	"syscall"
+	"testing"
 
 	"github.com/fsnotify/fsnotify"
 )
 
-func ExampleNewWatcher() {
+// If you want to execute example code, append prefix 'Test'.
+// ex) func TestExampleNewWatcher(t *testing.T)
+func exampleNewWatcher(t *testing.T) {
```

Comment:
The point of an example is to be code that people could apply to an application. It should be using testing.T.